### PR TITLE
JavaScript Capstone

### PR DESCRIPTION
**Pull request by mahdimahdawi**
**This PR Contain:**
- link of Card: https://github.com/mahdimahdawi/CapStone-project2/projects/1
- project setup with webpack.
- configure jest to test the function for unit testing.
- Added header and footer.
- Fetching data from API and display dynamically to UI using JavaScript.
- Show the total available number of movies.
- Post new likes to involvement API and Store the number of likes.
- Fetching the number of like and display to UI.
- When user like their favorite Movies the counter of like increment by one.
- The Recode link is also included to the README.md file.
### This PR Belong to @mahdimahdawi  and my partner did not push his code i reported to student success this issues they said they will unpair us.
Thanks in Advance.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  item.innerHTML = `
+  <img class="show-image" src="${imageUrl}" alt="${name}" />
+  <div class="name-like">
+  <h3 class="movie-name"> ${name} </h3>
+  <img class="like-image" src="${heartIcon}" alt="like" />
+  <p class="num-like"></P>
+  </div>
+  <p class="text-like">Like</P>
+  <div class="btns">
+  <button id="${id}" class="comment-btn btn">Comments</button>
```
Comment (MahmoudBakr23) [inline src/index.js:33]: - [ ] In the comment button on each of the movies' cards, it should display a popup menu with all the comments saved in an API such as the functionality of the likes. Please see the requirement [here](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=the%20given%20mockup.-,Comments%20popup,-When%20the%20popup).

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Great design
- Well done working with the API to fetch the movies and to POST and save the number of likes
- Nice job displaying the number of available movies in the header
- Good file structure
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️

- [ ] Please make sure to record a short 5 min video presenting your capstone project. 🙏 

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
Hi sir,
Hope your doing well.
As I mentioned on the PR description my coding partner did not push his
code to the repo. I done with my part that is displaying movie item and
like counter section. Please kindly review for Student A Card that I
mentioned the link on description.
Looking forward to hear from you.
Regards.
Mahdi Mahdawi.

**Comment by MahmoudBakr23** [comment]
Hi @mahdimahdawi
Thank you for contacting me back. I do get the situation from the beginning. Therefore, if student success has already unpaired you, then the next code reviewer should approve your project. However, if you're on this project solo from the very start, I guess you'll still have to wait for a pair up or choose to do it all on your own!
Anyways, it's always good to dig into more features to practice coding even more 😉

**Pull request closed without merging**